QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,135 @@
+This document is a declaration of software quality for the `class_loader` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `class_loader` Quality Declaration
+
+The package `class_loader` claims to be in the **Quality Level 1** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`class_loader` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning), and is at or above a stable version. **The current version is 1.4.0**.
+
+### Version Stability [1.ii]
+
+### Public API Declaration [1.iii]
+
+All symbols in the installed headers are considered part of the public API.
+
+All installed headers are in the `include` directory of the package, headers in any other folders are not installed and considered private.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`class_loader` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`class_loader` will maintain ABI stability within a ROS distribution.
+
+## Change Control Process [2]
+
+`class_loader` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change_control_process).
+
+### Change Requests [2.i]
+
+All changes occur through a pull request
+
+### Contributor Origin [2.ii]
+
+Must have confirmation of contributor origin (e.g. DCO, CLA, etc.).
```

Comment:
This should be rephrased to state what DCO process it follows, not just repeat the info in rep-2004.